curated_transformers/tokenizers/tokenizer.py

```diff
@@ -74,7 +74,7 @@ def padded_tensor(
         device: Optional[torch.device] = None,
     ) -> Tensor:
         """
-        Padded CPU tensor of the piece identifiers.
+        Generate a padded tensor of the piece identifiers.
```

Comment:
Maybe we should add a note here that the actual padding id typically doesn't matter when an attention mask is used, as long as the id is a valid vocabulary index? As a user I would be worried that I have to give some padding identifier, but how do I know which one?

On that note, we could consider just giving it a default value of `0` as well.

Reply:
Done.